recipes/next/app/[...puckPath]/page.tsx

```diff
@@ -28,7 +28,7 @@ export async function generateMetadata({
   }
 
   return {
-    title: getPage(path).root.title,
+    title: getPage(path)?.root.title,
```

Comment:
Ah great catch! Can you also update the generator template [here](https://github.com/measuredco/puck/blob/main/packages/create-puck-app/templates/next/app/%5B...puckPath%5D/page.tsx#L31)?

Unfortunately these are currently duplicated (see #10)